a10_octavia/controller/housekeeping/house_keeping.py

```diff
@@ -96,3 +94,21 @@ def cleanup_load_balancers(self):
             LOG.info('Attempting to delete load balancer id : %s', lb_id)
             self.lb_repo.delete(session, id=lb_id)
             LOG.info('Deleted load balancer id : %s', lb_id)
+
+
+class WriteMemory(object):
+    def __init__(self):
+        self.thunder_repo = a10repo.VThunderRepository()
+        self.cw = cw.A10ControllerWorker()
+
+    def perform_memory_writes(self):
+        exp_age = datetime.timedelta(seconds=CONF.a10_house_keeping.write_mem_expiry_time)
```

Comment:
Our goal is to do write memory for vThunders that has recently provisioned.
So, all vthunders which provisioned later than latest perform_memory_writes() called should do the write memory process. So, I think the age is (write_mem_interval + <time used by perform_memory_writes() latest time>)